tests/installation/accept_license.pm

```diff
@@ -25,7 +25,15 @@ sub run {
     my ($self) = @_;
     assert_screen 'license-agreement';
     $self->verify_license_has_to_be_accepted;
-    send_key $cmd{next};
+    wait_screen_change { send_key $cmd{next} };
+
+    # workaround bsc#1058503 - license shown twice on s390x
+    if (check_var('ARCH', 's390x') && check_screen('license-agreement-accepted', 5)) {
```

Comment:
Let's maybe invert logic here, so we check that it's not next screen. We will waste timeout unless fixed, but I guess it removes risks of recognizing same screen twice, as wait_screen_change may simply time out. Otherwise LGTM

Reply:
well `license-agreement-accepted` is a new needle which I created only for that purpose, so it should not recognize the same screen twice

Reply:
I assumed timeout of 0 should just work because we have a wait_screen_change above? But thinking about it while I write I think timeout of 0 will not work because the installer page might still be changing.